distribution/docs/src/main/resources/_contents/_data/media-attributes-table-contents.adoc

```diff
@@ -72,18 +72,20 @@ STANAG 4559 NC, NM, C1, M1, I1, C3, M3, C4, M4, C5, M5, C8, M8
 |A two-part identifier for file formats and format content
 |String
 |a valid https://en.wikipedia.org/wiki/Media_type[mime-type]
+TODO mime-type is defined in SIX linked RFC specs, but wikipedia isn't really authoritative
 |application/json
 
 |media.encoding
 |The encoding format of the media.
-|String
-|< 1024 characters
+|List of Strings
+|< 1024 characters per entry
+TODO this needs to be more well-defined
 | 
  
 |media.number-of-bands
 |The number of spectral bands in the media.
 |Integer
-| 
+| TODO spec reference?
```

Comment:
Wikipedia has a rough [categorization](https://en.wikipedia.org/wiki/Multispectral_image#Classification). I believe STANAG went from 1-9? I would suggest taking inspiration from the two and making our normal form an enumerated value set for clarity. What do the integers in STANAG map to? Frequency ranges?